src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitationsQueryBase.cs

```diff
@@ -179,30 +162,5 @@ private static string GetIpoIdStartWith(string filterString)
                 return null;
             }
         }
-
-        private static string NameCombiner(Participant participant) 
```

Comment:
why duplicate these methods in "child" classes instead of having them here in base?
You can make them protected and use them as it was